docs/DebuggingNumericalError.md

```diff
@@ -33,10 +33,11 @@ reference inputs and outputs in protobuf.
 ```bash
 $ python ../utils/RunONNXModel.py  --help
 usage: RunONNXModel.py [-h] [--print_input] [--print_output]
-                       [--save_so PATH | --load_so PATH] [--save_data PATH]
+                       [--save_onnx PATH | --save_so PATH | --load_so PATH]
+                       [--save_data PATH]
                        [--data_folder DATA_FOLDER | --shape_info SHAPE_INFO]
                        [--compile_args COMPILE_ARGS]
-                       [--verify {onnxruntime,ref}]
+                       [--verify {onnxruntime,ref}] [--verify_all_ops]
```

Comment:
if you push a new version, maybe each option on a separate line

Reply:
Done. Thanks!